.github/workflows/containers.yml

```diff
@@ -22,8 +22,18 @@ jobs:
 
       - name: Publish Containers
         run: |
+          versions=("latest")
           GIT_VERSION=$(git describe --tags --always --match 'v*')
-          docker tag topo-imagery ghcr.io/linz/topo-imagery:${GIT_VERSION}
-          docker push ghcr.io/linz/topo-imagery:${GIT_VERSION}
-          docker tag topo-imagery ghcr.io/linz/topo-imagery:latest
-          docker push ghcr.io/linz/topo-imagery:latest
+          if [ ${{(github.ref == 'refs/heads/master' && contains(github.event.head_commit.message, 'release:'))}} == true ]; then
```

Comment:
Have to use contains as when the `release-please` PR gets merged, the commit message not only contains the `release: ...` commit:
```
commit 68f58f69bbaec5a5895a1ef3572949ec516ac39a (HEAD -> master, tag: v0.4.0, origin/master, origin/HEAD)
Merge: 94fb2a7 1ca37f3
Author: paulfouquet <86932794+paulfouquet@users.noreply.github.com>
Date:   Mon Jan 16 10:54:41 2023 +1300

    Merge pull request #29 from paulfouquet/release-please--branches--master
    
    release: 0.4.0
```

Reply:
your commit  here looks to be a Merge commit, we do not allow merge commits in this repo.

When merging it should be merged as a Squash+Rebase, here is an example commit with a squash/rebase

https://github.com/linz/style-js/commit/169ad32dd68de237433e4079bca0ffe29f8f4569

so `startsWith` is what we want.

Reply:
I haven't do my test in a exact same environment that's why...